src/components/Home/ChatBot/Modal.jsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image";
+import ReactDOM from "react-dom";
+import Caramelo from '@/assets/cachorroHeaderChatBot.png';
+import '../../../app/globals.css';
+import './Styles/iconStyle.css';
+
+
+const Modal = ({ onClose, children }) => {
+
+    const handleCloseClick = (e) => {
+        e.preventDefault();
+        onClose();
+    }; {/*Função que armazena o evento de clique para fechar o modal */}
+
+    const modalContent = (
+        <div className="fixed teste top-0 left-0 w-full h-[100%] flex justify-center items-center"> {/*Modal overlay*/}
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" /> {/*Api do google, para importar icones */}
+
+            <div className="fixed flex backdrop-opacity-10 rounded-2xl bottom-35 right-11 w-[500px] h-[500px]"> {/*Modal wrapper*/}
+            
+                <a className="justify-self-start place-self-start" href="#" onClick={handleCloseClick}>
+                    <span class="material-symbols-outlined">
+                        cancel
+                    </span>
+                </a>  {/*Modal Botão de fechar*/}
```

Comment:
O botão de fechar está posicionado no local errado, porém é um ajuste simples de ser feito, inclusive já fiz isso na estilização, caso queira pode testar o código abaixo:

<div>
                            <div>
                                <header className=" flex rounded-t-lg flex-nowrap w-[500px] h-16 bg-gradient-to-r from-[#A110A9] to-[#33B3A6]"> {/*Modal Cabeçalho*/}

                                <div className="w-[14%]"> {/*Imamgem caramelo*/}
                                    <Image className=" w-auto h-full rounded-t-lg" src={Caramelo} alt="Imagem do mascote da plataforma, caramelo, olhando para o horizonte." />
                                </div>


                                <div className="w-[80%] text-center items-center self-center text-white	font-bold
                                ">
                                    <h1 className="text-2xl tracking-wide">Caramelo irá te ajudar!</h1>
                                </div>                              
                                
                                <div className="">
                                    <a className=" relative left-[8px] bottom-[2px]" href="#" onClick={handleCloseClick}>
                                    <span class="material-symbols-outlined">
                                    cancel
                                    </span>
                                    </a>  {/*Modal Botão de fechar*/}
                                </div>
                                </header>
                            </div>
                        </div>